include/forcing/NullForcingProvider.hpp

```diff
@@ -0,0 +1,137 @@
+#ifndef NGEN_NULLFORCING_H
+#define NGEN_NULLFORCING_H
+
+#include <vector>
+#include <set>
+#include <cmath>
+#include <algorithm>
+#include <string>
+#include <fstream>
+#include <iostream>
+#include <unordered_map>
+#include <boost/algorithm/string.hpp>
+#include <boost/lexical_cast.hpp>
+#include <ctime>
+#include <time.h>
+#include <memory>
+#include "AorcForcing.hpp"
+#include "GenericDataProvider.hpp"
+#include "DataProviderSelectors.hpp"
+#include <exception>
+#include <UnitsHelper.hpp>
+
+/**
+ * @brief Forcing class providing time-series precipiation forcing data to the model.
```

Comment:
Update this doc string to reflect that this forcing provider does not, in fact, provide any forcing data!

Reply:
I am on it.